config.go

```diff
@@ -102,6 +105,11 @@ type config struct {
 	HTTP2Downstream      bool
 
 	flags *flag.FlagSet
+
+	//!
```

Comment:
Special comments to mark where your changes are should not be needed now that we're tracking the change with git.

It would probably be cleaner to move these lines up a little and leave `flags` as the last field of config.